src/hooks/jitPluginInstall.ts

```diff
@@ -17,11 +17,11 @@ const hook: Hook<'jit_plugin_not_installed'> = async function (opts) {
       eventName: 'JIT_INSTALL_STARTED',
       type: 'EVENT',
       version: opts.pluginVersion,
-      plugin: opts.command.pluginName,
+      plugin: opts.pluginName,
       command: opts.command.id,
     });
 
-    const jitInstallArgv = [`${opts.command.pluginName}@${opts.pluginVersion}`];
```

Comment:
opts.command says it can be undefined.  
this hook's opts include a `pluginName` prop